### PR TITLE
[SL Alpha5] Add giphy connector

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
JDK 11
Ballerina Alpha5
Ubuntu 20.04

### CODE REVIEW

File: openapi/giphy/client.bal [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import  ballerina/http;
```
Comment (indikasampath2000) [inline openapi/giphy/client.bal:1]: Missing license header

File: openapi/giphy/types.bal [hunk 2]
```diff
@@ -0,0 +1,135 @@
+public type Response record {
```
Comment (indikasampath2000) [inline openapi/giphy/types.bal:1]: Missing license header

Comment (Chuhaa) [reply openapi/giphy/types.bal:17]: Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fcd3580d6892ae0cfbfa1e9449cc814999a11f4b

File: openapi/giphy/client.bal (hunk 1 above)
Comment (Chuhaa) [reply openapi/giphy/client.bal:17]: fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fcd3580d6892ae0cfbfa1e9449cc814999a11f4b

**Review by indikasampath2000 — APPROVED** [review]

**Pull request merged**